packages/compass-e2e-tests/index.js

```diff
@@ -35,7 +35,10 @@ async function main() {
     mocha.addFile(path.join(__dirname, testPath));
   });
 
-  mocha.run((failures) => (process.exitCode = failures ? 1 : 0));
+  mocha.run((failures) => {
+    process.exitCode = failures ? 1 : 0;
+    process.exit();
```

Comment:
Nothing happens after that run, so calling exit should not be necessary I think

```suggestion
```

Reply:
CI wasn't resolving/exiting when tests failed without it. Tried to see what was causing it to hang but couldn't find this issue, this change ensured it still exited on failures.

Reply:
Ah, okay, thanks for the context!